modules/rubiconBidAdapter.js

```diff
@@ -22,6 +24,10 @@ const VIDEO_ENDPOINT = '//fastlane-adv.rubiconproject.com/v1/auction/video';
 
 const TIMEOUT_BUFFER = 500;
 
+var ACCOUNT_ID;
+getConfig('rubiconAdapter', config => ACCOUNT_ID = config.accountId);
```

Comment:
I don't know if I'd call it a race condition, but there is definitely a dependency problem.  However, I didn't create it.  The problem is inherent in the way we execute `pbjs.que` blocks and when module code is ran by webpack.  I detailed this in a diagram I pasted to slack awhile back to describe our problem:
<img width="180" alt="screen_shot_2017-05-31_at_1 09 58_pm_480" src="https://user-images.githubusercontent.com/551085/26951694-5d8996ca-4c5f-11e7-9afd-65d9b7f4fb2f.png">

We effectively have a circular dependency based on the current architecture of Prebid.js.  This isn't a new problem, it's just now more obvious because this is the first time someone has attempted to apply global config (config across adUnits and auctions) to an adapter (or in this case now, a module).  Currently configuration can only come in through `bid.params` which makes this condition less obvious, but is also not an ideal solution for handling global config.

Using a callback was the solution I found to solve this.  By registering a callback that sets the config it allows us to defer configuration until after `setConfig` is called.  I think that dependency is intuitive (that you must `setConfig` for a module before you can use that config), so I'm not too worried about the implications.

I originally tried to solve the problem with promises, but then I ran into an _actual_ race condition when the scheduled config callback wasn't called until _after_ the auction had ran.

It's possible to not register the adapter to Prebid.js until the config has been received (the registration being the real export that Prebid cares about, `module.exports` is only used for testing), but then you're just swapping a set of config error messages, for a more cryptic error general message.  i.e. "Rubcion Adapter missing required config Account ID" vs "Rubicon Adapter was not found".